IncrementalSVDFastUpdate.C

```diff
@@ -341,22 +341,6 @@ IncrementalSVDFastUpdate::addNewSample(
       max_U_dim = d_total_dim;
    }
 
-   if (fabs(checkOrthogonality(d_U)) >
-       std::numeric_limits<double>::epsilon()*static_cast<double>(max_U_dim)) {
-      reOrthogonalize(d_U);
-   }
-   if (fabs(checkOrthogonality(d_Up)) >
```

Comment:
Hi Youngsoo. Here, d_U and d_Up are orthogonalized which ensure d_basis is orthogonalized afterwards. So when svtol is turned off, d_basis remain orthogonal. But when svtol is turned on, if a column is removed, d_basis is modified. I'm not too familiar with orthogonality, but it seems d_basis may not be orthogonal after a column is removed.